packages/common/package.json

```diff
@@ -0,0 +1,12 @@
+{
+  "name": "@wallet-service/common",
+  "packageManager": "yarn@4.1.0",
+  "peerDependencies": {
+    "@aws-sdk/client-lambda": "3.540.0",
+    "@hathor/wallet-lib": "0.39.0",
+    "winston": "^3.13.0"
```

Comment:
These are defined as `peerDependencies` so they use the same version from the root `package.json`